__tests__/index.test.js

```diff
@@ -120,6 +120,37 @@ describe("GET /api/articles", () => {
         expect(body.articles).toBeSortedBy("created_at", { descending: true });
       });
   });
+
+  test("status 200: responds with an array of article objects filtered by topic", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(12);
+        body.articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
+
+  test("status 200: responds with a message 'No articles found' when given a topic with no articles", () => {
+    return request(app)
+      .get("/api/articles?topic=paper")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.msg).toBe("No articles found");
```

Comment:
It's probably more appropriate for this to be a 'normal' response that happens to be empty. Any logic to present to the end user could be handled on the front end to communicate the lack of articles. I would suggest changing this to be just a test for the empty array.